steam_app/controllers/Games/GamesViewController.swift

```diff
@@ -9,11 +9,15 @@ import UIKit
 
 class GamesViewController: UIViewController {
     @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var search: UISearchBar!
     let gamesList = LocalCache.gamesList
+    var filteredData = [Game]()
```

Comment:
it's a bad approach to put an instance of the Game inside the array as the array will never be empty.
Better to put it this way var filteredData: [Game] = []